webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsPage.js

```diff
@@ -54,7 +54,7 @@ class UpstreamSubscriptionsPage extends Component {
               />
               <PaginationRow
                 viewType="table"
-                itemCount={upstreamSubscriptions.pagination.total}
+                itemCount={upstreamSubscriptions.itemCount}
```

Comment:
Why item count rather than using total from the pagination?

Reply:
I used `itemCount` in the state just because it's how the param is named in patternfly-react. I'm ok to rename it.
BTW note that we have to use `subtotal`, not `total`
(`total` = # of all records in the db vs. `subtotal` = # of all records matching your search).